migrations/20230830024631_cw721_activity_add_current_owner.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('cw721_activity', (table) => {
+    table.string('owner');
```

Comment:
để nguyên định nghĩa `from` với `to`, em thêm `sender` thì nó hợp lý hơn

Reply:
from và to hiện tại đang có data, thực chất from đang là data của sender, nếu thêm cột sender thì phải chuyển data từ from qua ( hoặc rename cột ). Em thêm cột owner thì chỉ cần fill data vào mà em thấy cũng tạm chấp nhận được, cái tên cột mình hiểu là đc mà :))

Reply:
😛